src/main.cpp

```diff
@@ -137,7 +136,7 @@ int main(int argc, char** argv)
     QObject::connect(&app, SIGNAL(openFile(QString)), &mainWindow, SLOT(openDatabase(QString)));
     QObject::connect(&app, SIGNAL(quitSignalReceived()), &mainWindow, SLOT(appExit()), Qt::DirectConnection);
 
-    Bootstrap::restoreMainWindowState(mainWindow);
+    mainWindow.restoreConfigState();
```

Comment:
We should just call this within MainWindow constructor and connect those qApp signals in there as well.

Reply:
So you'd pass the app in the `MainWindow`'s constructor?

Reply:
No use the global qApp macro